### ISSUE TITLE
EBS Snapshot Module

### ISSUE THREAD

**Issue by bpmct** [coder/modules#284]
Create a module that kicks off an EBS snapshot on workspace stop and allows workspaces to be optionally created/started  from a snapshot via a parameter. Otherwise, it will use the existing PVC/EBS volume/disk.

**Comment by michaelbrewer** [comment]
I assume this is because EBS snapshots are cheaper than an EBS volume? However the side effect might be slower start up?

**Comment by michaelbrewer** [comment]
Makes sense. Especially when you want to decommission a region, eks cluster or AWS account.
